docs/_includes/generated/environment_options_configuration.html

```diff
@@ -0,0 +1 @@
+<table class="table table-bordered"><thead><tr><th class="text-left" style="width: 20%">Key</th><th class="text-left" style="width: 15%">Default Value</th><th class="text-left" style="width: 65%">Description</th></tr></thead><tbody><tr><td><h5>env.java.opts</h5></td><td>(none)</td><td>Set custom JVM options. This value is respected by Flink's start scripts, both JobManager and TaskManager, and Flink's YARN client. This can be used to set different garbage collectors or to include remote debuggers into the JVMs running Flink's services. Enclosing options in double quotes delays parameter substitution allowing access to variables from Flink's startup scripts.&#60;p&#62;Use `env.java.opts.jobmanager` and `env.java.opts.taskmanager` for JobManager or TaskManager-specific options, respectively. &#60;/p&#62;</td></tr><tr><td><h5>env.java.opts.jobmanager</h5></td><td>(none)</td><td>JobManager-specific JVM options. These are used in addition to the regular `env.java.opts`.</td></tr><tr><td><h5>env.java.opts.taskmanager</h5></td><td>(none)</td><td>TaskManager-specific JVM options. These are used in addition to the regular `env.java.opts`.</td></tr></tbody></table>
```

Comment:
But i would just add a new-line after each tag, so that it looks like this:
```
<table class="table table-bordered">
<thead>
<tr>
<th class="text-left" style="width: 20%">Key</th>
<th class="text-left" style="width: 15%">Default Value</th>
<th class="text-left" style="width: 65%">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><h5>blob.fetch.backlog</h5></td>
<td>1000</td>
<td></td>
</tr>
<tr>
<td><h5>blob.fetch.num-concurrent</h5></td>
<td>50</td>
<td></td>
</tr>
<tr>
<td><h5>blob.fetch.retries</h5></td>
<td>5</td>
<td></td>
</tr>
<tr>
<td><h5>blob.server.port</h5></td>
<td>"0"</td>
<td></td>
</tr>
<tr>
<td><h5>blob.service.ssl.enabled</h5></td>
<td>true</td>
<td></td>
</tr>
<tr>
<td><h5>blob.storage.directory</h5></td>
<td>(none)</td>
<td></td>
</tr>
</tbody>
</table>
```